index.js

```diff
@@ -106,6 +80,8 @@ function FastForEach(spec) {
   this.as = spec.as;
   this.noContext = spec.noContext;
   this.noIndex = spec.noIndex;
+  // allow users to turn off the 'reuse/move DOM nodes' feature
+  this.allowMoveNodes = spec.allowMoveNodes === undefined ? true : spec.allowMoveNodes;
```

Comment:
I have no idea, I've added it to be able to run performance comparison tests more clearly. After that, I kept it there, why not. Probably there are some special scnearios and use cases out there what we can't support, or something like this. But can be deleted I think, if you don't like it :)